repository.yaml

```diff
@@ -1,5 +1,4 @@
-digital-service: BTA
```

Comment:
think this got added in by mistake or maybe was from the past - not sure if platui always had help-frontend

Reply:
😮